vita3k/host/dialog/src/filesystem_nfd.cpp

```diff
@@ -171,10 +170,9 @@ Result pick_folder(std::filesystem::path &resulting_path, const std::filesystem:
     } arguments;
 
     // If the default path isn't empty, specify the pointer to the default path string
-    const std::u8string default_path_u8 = default_path.u8string();
-    if (default_path != "") {
-        arguments.defaultPath = reinterpret_cast<const nfdu8char_t *>(default_path_u8.c_str());
-    }
+    const std::string default_path_str = default_path.string();
```

Comment:
It's possible to  have bugs with non ascii character paths here. (At least for windows versions before win10)

Reply:
I rechecked. Here u8string mean converting path to UTF-8 string. So unicode bug will happens here.